frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/vm-settings-tab-initial-state.ts

```diff
@@ -59,6 +59,7 @@ export const getInitialVmSettings = (common: CommonData) => {
       isRequired: asRequired(true),
     },
     [VMSettingsField.START_VM]: {
+      value: false,
```

Comment:
I don't have anything against this change. I don't really understand why this would fix that though. Do you know what was the underlying problem and why it worked in VMs?

It seems it should work just fine  judging by our request logic

```js
vm.spec.running = getSetting(START_VM_KEY, false);
```

Reply:
hmm ... yes, this should be redundant, but it does fix the bug on my setup ...
strange, will look what is happening.

Reply:
@suomiy hi, found it :-)
in our `getSettings` we do not accept default value :-)

https://github.com/kubevirt/web-ui-components/blob/master/src/k8s/request.js#L127

so if no value is defined it set into `undefined` , and not into the default value ...

this is the only place we use `getSetting` with a default value ... so I think it's safe to fix it on the `console` form side.

on the `web-ui-componenets` side we can:
a - remove the default value from line:
https://github.com/kubevirt/web-ui-components/blob/master/src/k8s/request.js#L377
because this is not how this function work.
b - add an optional default value ... to the `getSettings` method ?

Reply:
> Do you know what was the underlying problem

`getSettings` ignore the default value `false` and return `undefined`

> and why it worked in VMs?

In VMs this field have a value either `true` or `false`.

Reply:
great job figuring it out!